### DISCUSSION TITLE
rdk_doc/en/FAQ/hardware_and_system

### DISCUSSION THREAD

**Discussion by giscus** [D-Robotics/rdk_doc#59] [General]
# rdk_doc/en/FAQ/hardware_and_system

For certified accessories and purchasing links, please refer to the Certified Accessories List.

https://d-robotics.github.io/rdk_doc/en/FAQ/hardware_and_system

**Comment by citrahas** [comment]
Hi, is there any ways to force reboot the system without command? I accidentally unplug the power supply without command "sudo poweroff" and now no display after power on the development board.    I haven`t set the remote or ssh login

**Reply by xuguosheng110** [reply]
hello, dear citrahas , you`d better close system by command terminal or by desktop command, force shutdown may damage your root files
